external-crates/move/crates/move-compiler/tests/move_check/unit_test/other_failures_invalid_location@unit_test.snap

```diff
@@ -27,46 +27,46 @@ error[E10004]: invalid usage of known attribute
+
+error[E03002]: unbound module
+   ┌─ tests/move_check/unit_test/other_failures_invalid_location.move:26:47
    │
 26 │     #[expected_failure(vector_error, location=x)]
-   │                                      ^^^^^^^^^^
-   │                                      │        │
-   │                                      │        Expected a module identifier, e.g. 'std::vector'
-   │                                      Invalid value in attribute assignment
+   │                                               ^ Unbound module alias 'x'
 
-error[E10003]: invalid attribute value
-   ┌─ tests/move_check/unit_test/other_failures_invalid_location.move:30:36
+error[E03006]: unexpected name in this position
+   ┌─ tests/move_check/unit_test/other_failures_invalid_location.move:30:53
    │
 30 │     #[expected_failure(out_of_gas, location=0x1::m::t0)]
-   │                                    ^^^^^^^^^^^^^^^^^^^
-   │                                    │        │
-   │                                    │        Expected a module identifier, e.g. 'std::vector'
-   │                                    Invalid value in attribute assignment
+   │                                                     ^^ Unexpected module member access. Expected a module identifier only
 
-error[E10003]: invalid attribute value
-   ┌─ tests/move_check/unit_test/other_failures_invalid_location.move:34:43
+error[E03002]: unbound module
+   ┌─ tests/move_check/unit_test/other_failures_invalid_location.move:34:52
    │
 34 │     #[expected_failure(major_status=4004, location=self)]
-   │                                           ^^^^^^^^^^^^^
-   │                                           │        │
-   │                                           │        Expected a module identifier, e.g. 'std::vector'
-   │                                           Invalid value in attribute assignment
+   │                                                    ^^^^ Unbound module alias 'self'
+
+error[E10004]: invalid usage of known attribute
+   ┌─ tests/move_check/unit_test/other_failures_invalid_location.move:38:24
+   │
+38 │     #[expected_failure(major_status=4016, minor_status=0, location=0)]
+   │                        ^^^^^^^^^^^^^^^^^ Expected 'location' following 'major_status'
 
-error[E10003]: invalid attribute value
+error[E02015]: invalid attribute
    ┌─ tests/move_check/unit_test/other_failures_invalid_location.move:38:59
    │
 38 │     #[expected_failure(major_status=4016, minor_status=0, location=0)]
-   │                                                           ^^^^^^^^^^
-   │                                                           │        │
-   │                                                           │        Expected a module identifier, e.g. 'std::vector'
-   │                                                           Invalid value in attribute assignment
+   │                                                           ^^^^^^^^ Field 'location' must be a module name
```

Comment:
```suggestion
   │                                                           ^^^^^^^^ Field 'location' must be a module identifier
```

Reply:
This is generated by expansion, and is consistent with other test outputs. Let me know if we should change this across the test cases.